samples/bluetooth/peripheral_hr_coded/README.rst

```diff
@@ -70,7 +72,7 @@ After programming the sample to your development kit, you can test it by connect
 #. Wait until the Coded advertiser is detected by the Central.
    In the terminal window, check for information similar to the following::
 
-      Connected: xx.xx.xx.xx.xx.xx (random), tx_phy 4, rx_phy 4
+      Connected: xx.xx.xx.xx.xx.xx (random), tx_phy LE Coded, rx_phy LE Coded
```

Comment:
Would be nice if you could update the docs to also mention the log line "Filters matched..."